project/Build.scala

```diff
@@ -207,11 +207,28 @@ object DottyBuild extends Build {
     ).
     settings(publishing)
 
+  // until sbt/sbt#2402 is fixed (https://github.com/sbt/sbt/issues/2402)
+  lazy val cleanBridge = TaskKey[Unit]("clean-sbt-bridge", "delete dotty-sbt-bridge cache")
```

Comment:
Sorry if I wasn't totally clear, I meant that the `val` should be called `cleanSbtBridge`, but I'm fine with changing the key name too :)